shared/components/StatusBadge/StatusBadge.js

```diff
@@ -125,8 +125,7 @@ export const StatusBadge = ({
 
   if (additionalContent) {
     // Remove the dot at the end of the sentence if we add a colon afterwards
-    if (content?.endsWith('.')) content = content?.slice(0, -1);
-    content = `${content}: ${additionalContent}`;
+    content = additionalContent;
```

Comment:
are we sure `content` will always be the same as `badgeContent`? maybe checking `if (content === badgeContent)` would work better?

also after the changes, the comment in line 127 doesn't make sense